### PR TITLE
nef 0.6.1

### PR DESCRIPTION
**Pull request by miguelangel-dev**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

This pull request updates the nef formula and updates the requirements:
- Xcode 11.4+

### CODE REVIEW

**Comment by Bo98** [comment]
> Xcode 11.4+
> macOS 10.14+

Xcode 11.4 isn't available on Mojave.

**Comment by Bo98** [comment]
> How can I force to only run the tests in macOS >= 10.15?

The Xcode 11.4 implies 10.15 only. The 10.13 and 10.14 tests will still appear as "run" in GitHub, but it will skip building this formula when it sees the Xcode 11.4 requirement.

File: Formula/nef.rb [hunk 1]
```diff
@@ -1,16 +1,10 @@
...
-  sha256 "a67bb4201739898832ec52e92d838ccfcae25a490df433cd7cf44f1fc6e0a786"
+  url "https://github.com/bow-swift/nef/archive/0.6.1.tar.gz"
+  sha256 "182d5f91e3048f046a63bf81d148f94acc7c667b5314cf76f36a9fb75efdb3a7"
 
-  bottle do
-    cellar :any_skip_relocation
-    sha256 "4dffdd30d80ab45cda870e0e5ebe981f64ab878de8cabaeb13bb33cc5ef6d1a0" => :catalina
-    sha256 "49c50ed996365fa2636500525c5ab74554eb530afde00e94aaf004c2032fcf57" => :mojave
-  end
-
```
Comment (Bo98) [inline Formula/nef.rb:12]: You don't need to touch the bottle block. This is automatically managed when merging.

**Comment by miguelangel-dev** [comment]
@Bo98 any idea what is failing in CI?

**Comment by Bo98** [comment]
Network blip presumably. I'll rerun it.

**Review by Bo98 — APPROVED** [review]
Thanks!

**Pull request closed without merging**